### DISCUSSION TITLE
Not showing in absolute positioned div

### DISCUSSION THREAD

**Discussion by jrtp** [mdbassit/Coloris#148] [General]
Hi, lovely lib! Found one issue that for some reason a text input with data-coloris within an absolute positioned div does not show the picker when clicked. Also I was looking for a simple way to "cancel" so it goes back to the original color - is this possible somehow?

**Comment by mdbassit** [comment]
> Hi, lovely lib! Found one issue that for some reason a text input with data-coloris within an absolute positioned div does not show the picker when clicked.

It's hard for me to tell what the problem is without looking at your code.
Did you maybe set the `parent` option to this div as well? If so, does the div have enough room to display the picker?

> Also I was looking for a simple way to "cancel" so it goes back to the original color - is this possible somehow?

Pressing the `ESC` key will revert the color to its original value.

**Comment by jrtp** [comment]
Sorry, user error - not showing was somehow related to flex box, works like a charm now! Regarding ESC that would be the only thing I could wish for, that this option was somehow also available in UI.

**Reply by mdbassit** [reply]
Check out my answer in this issue https://github.com/mdbassit/Coloris/issues/123#issuecomment-1806687573
